lib/registries/addon/drafts/draft/form/route.ts

```diff
@@ -0,0 +1,7 @@
+import Route from '@ember/routing/route';
+
+export default class DraftsFormRoute extends Route.extend({}) {
```

Comment:
`.extend` shouldn't be necessary:
```suggestion
export default class DraftsFormRoute extends Route {
```